### ISSUE TITLE
Sunset

### ISSUE THREAD

**Issue by levino** [withfig/fig#2934]
I am sad to hear that fig is sunsetting. However I am wondering, since this is supposed to be an open source project and is under the MIT license, why it does not just continue as a community project? Is nobody willing to take over as a maintainer? Is the provisioning of expensive server resources necessary to enable fig?

**Comment by SupianIDz** [comment]
By removing the Plugin, Dotfiles, and Server features, I see this as a step backwards. Users are only being offered AI, which doesn't benefit those who heavily rely on dotfiles, servers, and plugins.

**Comment by levino** [comment]
I also really do not like the idea of all my prompts being sent to amazon. But that is another story. I do not want to criticise the new product. I hope that I can use fig also in the future. What kind of server infrastructure is needed? Can this requirement be removed without breaking fig?

**Comment by anthonyhexium** [comment]
That's such a shame to shutdown a project like this. CodeWhisperer has nothing similar to Fig. Fuck.

**Comment by SupianIDz** [comment]
I have no problem with my prompt being sent to Amazon, but my biggest question is, Why doesn't Amazon continue Fig instead of creating a new product and just taking autocomplete from Fig and leaving everything else. Even though in Fig previously it was planned that there would be AI.

**Comment by levino** [comment]
Even though I understand a certain degree of frustration, I think it is not worth while challenging the decision of the fig team.

Lets focus on the future of fig. It is under MIT license (at least as far as I know, is this true for all the code base?) and as such can never really be shut down. One just needs to find people who continue the maintenance and potentially the hosting for the back end (is there any which is really necessary for the CLI to work?).

**Comment by michealroberts** [comment]
Is anyone interested in forking this product and keeping it alive by other means? I have been boycotting Amazon and Amazon Web Services for their working and employment practises across other divisions, and total disregard of unionised workforces.

**Comment by mickaelxd** [comment]
I use the customize terminal feature a lot. Before 2021 I had a lot of trouble configuring the terminal "by hand", and with fig it was all like magic. I faithfully hope that someone manages to keep Fig online, perhaps by creating an OpenFig. I would certainly be one of the testers.

**Comment by pfurini** [comment]
Unfortunately I don't think all the fig codebase is under MIT license. I can only find the autocomplete repo, that incidentally  is the only feature being supported by Amazon. 
Everything else I think it's closed source, and it will remain so unless the creators decide to release them under an OSS license.

**Comment by mostafa6765** [comment]
> I am sad to hear that fig is sunsetting. However I am wondering, since this is supposed to be an open source project and is under the MIT license, why it does not just continue as a community project? Is nobody willing to take over as a maintainer? Is the provisioning of expensive server resources necessary to enable fig?

maybe it will continue fig [https://github.com/withthefig](https://github.com/withthefig)

**Comment by mostafa6765** [comment]
> Is anyone interested in forking this product and keeping it alive by other means? I have been boycotting Amazon and Amazon Web Services for their working and employment practises across other divisions, and total disregard of unionised workforces.

@michealroberts https://github.com/withthefig

**Comment by wheresfrank** [comment]
> CodeWhisperer for command line has the same core features as Fig

It depends on who you ask. The core features for me are scripts and dotfiles. Looks like I have until September to find an alternative. Sad news.

**Comment by duckpuppy** [comment]
It's a real bummer for those of us working in environments where GenAI tools are not allowed on developer machines given the issues with licensing of output.

**Comment by cpalafox** [comment]
Installed Codewhisperer to try it out and it's not as responsive and not what i want. Such a disappointment. This is not the tool i wanted. What a waste :(. Glad the creators were able to get the bag tho, i guess

**Comment by irsptech** [comment]
How can we use Plugins in Code Whisper? My Fig setup is full of plugins and I don't believe that they will just drop support. This is so disrespectful with the community.

**Comment by davidldennison** [comment]
Same core features? That statement is actually outlandish. Lol

**Comment by michealroberts** [comment]
@brendanfalk Disappointed, but glad you made $$$. You could literally make a similar tool, right?

**Comment by o6uoq** [comment]
> @brendanfalk Disappointed, but glad you made $$$. You could literally make a similar tool, right?

Yes, I think _we_ (the community) should. You could use GitOps as the backing database vs parsing data through a remote server, and features and solutions that _we_ want and not the [MegaCorp](https://aws.amazon.com/).

**Comment by EloB** [comment]
It's insane what money can do. Did AWS bought the product just to put it down? I don't want anything todo with AWS. They are terrible at software.

**Comment by o6uoq** [comment]
[Why Not Fork The](https://github.com/withfig/fig/issues/2937)

**Comment by DarioLopes** [comment]
Eurf... we can't have nice things anymore, who asked for AI in a prompt... just install CodeWhisperer without knowing that it won't support scripts (and likely never will), so gone are my scripts. **Will the program simply stop working and become unusable after september 1**? If not Amazon can keep himself is CodeWhisperer.

**Comment by pbr0ck3r** [comment]
As someone who contributed to Fig, parts of Fig are open source (https://github.com/withfig/plugins, https://github.com/withfig/autocomplete) but the backend servers (used for auth and storing other fig related user data) and desktop application/apis itself were closed source. Fig chose not to sell user data or charge out of the gate, and thus took the funding path to pay the developers/backend costs. Naturally when a company takes funding (Y Combinator’s S20 class) that path then is naturally being acquired or adding a paid tier (such as teams/enterprise features) as its exit.

[Source to original Hacker News Announcement](https://news.ycombinator.com/item?id=27277819).

**Comment by DarioLopes** [comment]
Adios Fig, it was a good run, F Amazon.

**Comment by NeevJewalkar** [comment]
i dont wanna give my data to amazon for their ad revenue

**Comment by devpenzil** [comment]
Its really sad that Fig is Sunsetting. As a companion, I have to give a signoff. Adios Fig.
Gonna miss you ❤️

**Comment by grant0417** [comment]
Some good news here, we just open sourced the codebase for the desktop app and CLI here: https://github.com/aws/amazon-q-developer-cli
